### PR TITLE
[Serverless] Update code block format for commands

### PR DESCRIPTION
**Pull request by abrennan89**

### CODE REVIEW

File: modules/kn-trigger-list.adoc [hunk 1]
```diff
@@ -8,24 +8,33 @@
...
+[source,terminal]
 ----
-$ kn trigger list
 NAME    BROKER    SINK           AGE   CONDITIONS   READY   REASON
 email   default   svc:edisplay   4s    5 OK / 5     True
 ping    default   svc:edisplay   32s   5 OK / 5     True
 ----
 
-* To print a list of triggers in JSON format, enter the following command:
+[id="kn-trigger-list-json_{context}"]
```
Comment (sheriff-rh) [inline modules/kn-trigger-list.adoc:30]: one header/procedure per module: https://redhat-documentation.github.io/modular-docs/#module_anchor-and-file-names-concept

File: modules/odc-creating-apiserversource.adoc [hunk 2]
```diff
@@ -19,7 +19,7 @@
 
 .Verification steps
```
Comment (sheriff-rh) [inline modules/odc-creating-apiserversource.adoc:20]: Note that this module has two procedures as well.

File: modules/serverless-create-eventing-namespace.adoc [hunk 3]
```diff
@@ -37,6 +37,8 @@ image::create-eventing-namespace.png[Creating the `knative-eventing` namespace]
 .Procedure
```
Comment (sheriff-rh) [inline modules/serverless-create-eventing-namespace.adoc:37]: Two procedures

File: serverless/knative_eventing/serverless-kn-trigger.adoc [hunk 4]
```diff
@@ -65,8 +70,10 @@ include::modules/kn-trigger-describe.adoc[leveloffset=+1]
 .Procedure
```
Comment (sheriff-rh) [inline serverless/knative_eventing/serverless-kn-trigger.adoc:70]: Two procedures

**Review by sheriff-rh — APPROVED** [review]
Excellent work! I love the minimalist writing, and the code block formatting looks great! I made a couple comments about four modules that I found that have two sets of procedures or headers in them. While this doesn't affect this PR directly, just note that guidelines indicate one header and procedure per module. You can either tack that on to this PR if you like or file a new one.

Regardless, great work! I know this took some time and effort! LGTM! 🎉 🌮 🥇 

Message me if you have questions!

**Comment by abrennan89** [comment]
> You can either tack that on to this PR if you like or file a new one.

@sheriff-rh I think if it's OK with you I'll follow this up in another PR, just because this one is already so huge and I still have to CP it to other branches (I'll do this myself manually except for 4.5 just in case there are clashes). Thanks so much for the kind words and feedback!

**Pull request merged**